cli/package.json

```diff
@@ -60,7 +61,7 @@
     },
     "scripts": {
         "assets:prep": "rimraf assets && mkdir -p assets",
```

Comment:
Shouldn't the command be `mkdirp` here, since the `mkdirp` package is in the devDependencies, to not depend on the native mkdir?

Reply:
@ismay good catch!  Yeah, this should be `mkdirp` if we want to be able to build the platform tool on Windows.

Our `fs-extra` dependency is a runtime dependency and doesn't include a CLI, so I think `mkdirp` is still right here (unless we moved everything to `fs-extra-cli`)

Reply:
Changed to `mkdirp`

Reply:
> Our fs-extra dependency is a runtime dependency and doesn't include a CLI

Ah I assumed it did. Yeah sounds good 👍